components/rhs_thread/rhs_thread.jsx

```diff
@@ -394,6 +394,7 @@ export default class RhsThread extends React.Component {
                             <div
                                 ref='rhspostlist'
                                 className='post-right-comments-container'
+                                id='post-right-comments-container'
```

Comment:
Please use `camelCase`.  Also, I'd suggest ID to be
```suggestion
                                id='rhsPostList'
```